src/table/cells/handle-key-press.js

```diff
@@ -3,7 +3,20 @@ import { updateFocus } from './handle-cell-focus';
 
 const isCtrlShift = (evt) => evt.shiftKey && (evt.ctrlKey || evt.metaKey);
 
-export const updatePage = ({ evt, totalRowSize, page, rowsPerPage, handleChangePage, setShouldRefocus }) => {
+export const preventDefaultBehavior = (evt) => {
+  evt.stopPropagation();
+  evt.preventDefault();
+};
+
+export const handleWrapperKeyDown = ({
```

Comment:
how about renaming it to `handleTableWrapperKeyDown`

Reply:
sure

Reply:
done